content/games/factions/companies.md

```diff
@@ -41,6 +42,12 @@ Terrifying (2)
 Throwers (1)
 : If this Group does not already have a MI profile, add one at 6+ / 5+ / 6”.
 
+| Taunt                                     | Check | Requirements                                                                                                          | Effect                                                                                                                                                                                                                                                                                                |
+| :---------------------------------------- | :---: | :-------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Your Corpse Will Never Thaw!              |  6+   | Aurorans only. Must have failed to Cast a Spell on an enemy Group last turn.                                          | This Group must Cast at the target enemy Group if they are able. When Casting, this Group may reroll any dice. When activating or resolving any other Order, the enemy player may force this Group to reroll any dice.                                                                                |
+| We'll Trample Your Bones Like Fresh Snow! |  7+   | Hunters only. Must be within 12" of an enemy Group which caused a friendly Group to lose Fighting Strength last turn. | This Group becomes Reckless and Vicious against the target Group, +2" MO, and may only activate to Attack or Move toward the target Group until the target Group is Routed. If the target Group Routs, this Group must activate to Proclaim this Taunt against the nearest valid target, if possible. |
```

Comment:
do we want to specify in the last sentence that, if the group fails the taunt activation, they have to keep trying on subsequent activations until they succeed? might be unwieldy in an already-long bit of rules text, maybe best left unsaid (giving up a small bit of rules precision for readability).

Reply:
Oh, that's an interesting interpretation - the way I was _intending_ this to come across is that there's a one-off chance that they have gone berserk and have to keep going - they basically get a check after they finish an enemy Group - are they still bloodthirsty, or has their thirst been slaked?

Reply:
Oh okay, that makes sense. I'm happy leaving the last sentence as-is, in that case.